MediaAdminBundle/Manager/SaveMediaManager.php

```diff
@@ -81,10 +81,23 @@ public function getFilenameFromChunks(UploadedFile $uploadedFile)
      */
     public function isFileAllowed($filename)
     {
+        return in_array($this->getMimeType($filename), $this->allowedMimeTypes);
+    }
+
+    /**
+     * Return the file mime type
+     *
+     * @param $filename
+     *
+     * @return string
+     */
+    public function getMimeType($filename)
```

Comment:
This should be protected or added to the interface. I prefer to get this protected